starter-code/functions-and-arrays.js

```diff
@@ -1,16 +1,13 @@
-// Find the maximum
-function maxOfTwoNumbers (first, second) {
-
+//Encontrar el máximo de dos números
+function maxOfTwoNumbers(first, second){
+  return first > second ? first: second;
```

Comment:
I :heart: ternarios , casi lloro de emoción. SI me queréis terminar de hacer feliz...
- Fuera puntos y comas de final de línea
- Si los if tienen una sola línea en su interior, podemos pasar de los {}
Ayudad a contribuir a un mundo mejor!!